test/rules-ceddl-fullstory.spec.ts

```diff
@@ -88,29 +50,6 @@ describe('CEDDL to FullStory rules', () => {
     expectEqual(payload.promotion, 'LaborDay2020');
   });
 
-  it('it should convert strings to reals and send CEDDL cart properties to FS.event', () => {
```

Comment:
@ndonze-fs, just confirming, the remove of this test case is because it's more or less the same as `it should send CEDDL cart to FS.event`?  If so, I'm good with that.